supplier_product/test_views.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase, Client
+from django.urls import reverse
+from .models import SupplierProduct, Product, Supplier
+from django.contrib.auth.models import User
+import pytest
+
+
+@pytest.mark.django_db()
+class IncreaseQuantityTest(TestCase):
+    def setUp(self):
+        # Create a test client
+        self.client = Client()
+        # Create a test user
+        self.user = User.objects.create_user(username='testuser', password='testpassword')
+        self.supplier = Supplier.objects.create(supplier_account=self.user)
```

Comment:
Supplier missing business name

Reply:
Why would you use create method since there is save_supplier method that is created